src/Symfony/Bundle/FrameworkBundle/Command/MessageConsumeCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Message\Asynchronous\Transport\ReceivedMessage;
+use Symfony\Component\Message\MessageBusInterface;
+use Symfony\Component\Message\Transport\ReceiverInterface;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class MessageConsumeCommand extends Command
```

Comment:
Could we create a Worker/Loop in the component?

Reply:
Done in 9bf3d98296514f3654f0d62795e2575f683b8f20